onebusaway-android/src/main/java/org/onebusaway/android/ui/MyRemindersFragment.java

```diff
@@ -129,14 +138,29 @@ public void onDestroy() {
         super.onDestroy();
     }
 
+    private int currentSortOrder;
```

Comment:
Since these are member variables, could you please change to `mCurrentSortOrder`, etc.?